### ISSUE TITLE
Unexpected angular dependency for rails-assets-angular-ui-grid gem

### ISSUE THREAD

**Issue by dgertych-monterail** [tenex/rails-assets#265]
The things is that https://github.com/angular-ui/ng-grid/blob/master/bower.json 
don't set any angular dependency, but when I try to install with rails-assets, 
the gem requires specific `rails-assets-angular (<= 1.3, >= 1.2.16)` version

```
Bundler could not find compatible versions for gem "rails-assets-angular":
  In snapshot (Gemfile.lock):
    rails-assets-angular (= 1.3.15)

  In Gemfile:
    rails-assets-angular-ui-grid (= 3.0.0.rc.22) ruby depends on
      rails-assets-angular (<= 1.3, >= 1.2.16) ruby

    rails-assets-angular (~> 1.3.14) ruby
```

**Comment by Chatles** [comment]
I have the same issue. Waiting for answer...

**Comment by sheerun** [comment]
It should be fixed with newest stable version (3.0.1)

**Comment by dgertych-monterail** [comment]
@sheerun thx
